_pages/dev/styling-and-page-layout/directionality.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Directionality
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 2.1 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The directionality feature provides support for bidirectional text and layout. You can configure Spartacus to use a left-to-right (LTR) orientation, or a right-to-left (RTL) orientation.
+
+Directionality is driven by language. Many languages are read from left to right, but well known examples, such as Arabic and Hebrew, are read from right to left.
+
+In Spartacus, the direction of the UI reflects the active language, so that directionality can work in a multi-directional experience. If your storefront contains both LTR and RTL languages, the active language is used to detect the direction automatically.
```

Comment:
"bi-directionality" is a common wording, I'd prefer to stick to it. Loads of frameworks made the bidi abbreviation out of it.

Reply:
Yes, "bidirectional" makes much more sense than "multi-directional", in this case 😄